src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -393,6 +393,23 @@ export class TeamlistComponent implements OnInit, OnDestroy {
   editTeamWrapper() {
     const SELF = this;
     const editTeam = (team) => {
+      if (this.isHost) {
+        SELF.apiCall = (params) => {
+          const BODY = JSON.stringify(params);
+          SELF.apiService.patchUrl(SELF.endpointsService.HostTeamURL(team), BODY).subscribe(
```

Comment:
Here we can just update the url based on the condition instead of whole API call.

Reply:
Done